asset/asset.go

```diff
@@ -427,6 +430,23 @@ type GroupKey struct {
 	Sig schnorr.Signature
 }
 
+// GroupKeyReveal is a type for representing the data used to derive the tweaked
+// key used to identify an asset group. The final tweaked key is the result of:
+// TapTweak(groupInternalKey, tapscriptRoot)
+type GroupKeyReveal struct {
+	// RawGroupKey is the public key that is tweaked twice to derive the
+	// final tweaked group key. The final tweaked key is the result of:
+	// groupInternalKey =  RawKey * sha256(RawKey || assetID) * G.
```

Comment:
BIP was updated to have the assetID be first in the sha256: https://github.com/Roasbeef/bips/pull/33#issuecomment-1669247706